torch/csrc/jit/runtime/register_ops_utils.h

```diff
@@ -34,11 +34,17 @@
 
 namespace torch {
 namespace jit {
-c10::AliasAnalysisKind aliasAnalysisFromSchema();
+inline c10::AliasAnalysisKind aliasAnalysisFromSchema() {
+  return c10::AliasAnalysisKind::FROM_SCHEMA;
+}
 
-c10::AliasAnalysisKind aliasAnalysisConservative();
+inline c10::AliasAnalysisKind aliasAnalysisConservative() {
+  return c10::AliasAnalysisKind::CONSERVATIVE;
+}
 
-c10::AliasAnalysisKind aliasAnalysisSpecialCase();
+inline c10::AliasAnalysisKind aliasAnalysisSpecialCase() {
+  return c10::AliasAnalysisKind::INTERNAL_SPECIAL_CASE;
+}
```

Comment:
What's the reason to make them inline? I though it only returns a enum element.

Reply:
it's a simple enum return so inline should make it faster in C++, also it seems like `register_ops_utils.cpp` have some issue linking the symbols with the header, so if I don't use inline, there's a linking issue when we use these function in any op registration (didn't dig into it further).